### PR TITLE
Add Fabio Fumarola profile

### PR DESCRIPTION
**Pull request by fabiofumarola**
Fixes issue #456 

Brief description of changes proposed in this pull request: added my profile in italian/english and picture

cc @teamdigitale/code-reviewers

### CODE REVIEW

File: en/people/fabio-fumarola.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+medium_user:
+linkedin_url: http://linkedin.com/in/fabio-fumarola
+ref: fabio-fumarola
+parent_ref: team
+start_date: October 30, 2017
+period_provided: until September 16, 2018
+annual_compensation: € 90,000
+approved_by_court: -
+---
+I was eight years old when I started to program with a Commodore VIC-20 bought together with an English course from my parents, but it was all by chance. Finished high school, I decided to study computer science at University of Bari. At the beginning it was hard, but after that I literally fallen in love for this field of study. I got a first and second level degree in five years, in 2008 I received a prize as best graduate student for the Science Faculties at University of Bari, and decide to go for a PhD in Machine Learning.
```
Comment (gvarisco) [inline en/people/fabio-fumarola.md]: decide -> decided

File: en/people/fabio-fumarola.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+During this period I did research on Information Network Analysis with Tim Weninger and produced great research published on top ranked journal and conferences. In May 2011 I defended my thesis with supervisor Professor Donato Malerba. Got the Ph.D, I spent one year at the the Department of Biomedical Science and Human Oncology as research assistant, where I collaborated on the research to identify the genes that are modified by blood tumors. Simultaneously, I was research assistant at University of Bari working on Data Stream Mining applied to photo-voltaic plants. In 2013, I worked as Data Scientist at Mer.Mec. group and founded an Innovative Startup with the goal of applying Machine Learning algorithms to create map that helps in monitoring the quality of city neighborhoods. In 2014, I was research assistant at University of Bari working on Web Mining and Big Data.
```
Comment (gvarisco) [inline en/people/fabio-fumarola.md]: map -> a map
OR
map -> maps that help

File: en/people/fabio-fumarola.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+In 2015, I started a position as Data Scientist in Unicredit Research and Development. I spent almost three years there working on a lot of interesting stuffs ranging from Natural Language Processing, Graph Mining, Big Data, data encryption and High Frequency Trading. I was a great experience with wonderful and open minded researchers.
```
Comment (gvarisco) [inline en/people/fabio-fumarola.md]: started a position as -> worked as

Comment (gvarisco) [reply en/people/fabio-fumarola.md]: stuff is uncountable. Therefore: stuffs -> stuff

Comment (gvarisco) [reply en/people/fabio-fumarola.md]: I was a great experience -> It was a great experience

File: en/people/fabio-fumarola.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+During 2017, I heard about the Digital Transformation Team while I was in Unicredit. It was something that I followed with a lot of interest. Then, in late 2017 I found that they had an open position as Machine Learning Engineer and I sent my curriculum, and after several interviews I got hired. I'm very honored to work in the Digital Transformation Team under the guidance of Diego Piacentini. I'm sure that I'll learn a lot from him and from the team and to collaborate in driving innovation to my Country.
```
Comment (gvarisco) [inline en/people/fabio-fumarola.md]: if the main verb is 'learn', 'to collaborate in' doesn't sound good to me. I'd rephrase as follows:
I'm sure that I'll learn a lot from him and from the team, as well will get the chance to contribute in driving innovation to my Country.

File: en/people/fabio-fumarola.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+During 2017, I heard about the Digital Transformation Team while I was in Unicredit. It was something that I followed with a lot of interest. Then, in late 2017 I found that they had an open position as Machine Learning Engineer and I sent my curriculum, and after several interviews I got hired. I'm very honored to work in the Digital Transformation Team under the guidance of Diego Piacentini. I'm sure that I'll learn a lot from him and from the team and to collaborate in driving innovation to my Country. 
+
+I describe myself as passionate about using data to solve issues, to uncovering actionable insights and to create descriptive/predictive models to help teams and companies become more productive. I consider myself a communicative person that values building strong relationships with colleagues and stakeholders and have the ability to explain complex topics in simple terms. The characteristic that represents me the most is my intellectual curiosity and the desire to improve myself.
```
Comment (gvarisco) [inline en/people/fabio-fumarola.md]: to uncovering -> to uncover

**Comment by fabiofumarola** [comment]
@gvarisco please check if it is ok, thanks

File: en/people/fabio-fumarola.md [hunk 6]
```diff
@@ -20,7 +20,7 @@ I was eight years old when I started to program with a Commodore VIC-20 bought t
...
 
-In 2015, I started a position as Data Scientist in Unicredit Research and Development. I spent almost three years there working on a lot of interesting stuffs ranging from Natural Language Processing, Graph Mining, Big Data, data encryption and High Frequency Trading. I was a great experience with wonderful and open minded researchers.
+In 2015, I started a woked as Data Scientist in Unicredit Research and Development. I spent almost three years there working on a lot of interesting stuff ranging from Natural Language Processing, Graph Mining, Big Data, data encryption and High Frequency Trading. It was a great experience with wonderful and open minded researchers.
```
Comment (gvarisco) [inline en/people/fabio-fumarola.md]: wrong
